src/components/voteButtons.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState } from "react";
+import { patchReviewVotes } from "./api-requests";
+
+const VoteButtons = ({ voteAmount, review_id }) => {
+  const [votes, setVotes] = useState(voteAmount);
+  const [errMsg, setErrMsg] = useState(null);
+  const [hasVoted, setHasVoted] = useState(false);
+  const [voteInc, setVoteInc] = useState(null);
+
+  const handleClick = (e) => {
+    if (e.target.textContent === "+") {
+      setVoteInc(1);
+    } else if (e.target.textContent === "-") {
+      setVoteInc(-1);
+    }
+    if (!hasVoted) {
+      setHasVoted(true);
+      setVotes((currVotes) => currVotes + 1);
+      setErrMsg(null);
+      patchReviewVotes(review_id, { inc_votes: 1 }).catch((err) => {
```

Comment:
Currently, The vote is going up whether I press + or - Think you just need to tweak your `inc_votes` object slightly to fix this :)